scripts/docker-compose-testing.yml

```diff
@@ -51,6 +53,16 @@ services:
     volumes:
       - mysqldata:/var/lib/mysql
 
+  sqlserver:
+    image: mcr.microsoft.com/mssql/server:2022-latest
+    environment:
+      - MSSQL_PID=Developer
+      - ACCEPT_EULA=Y
+      - MSSQL_SA_PASSWORD=Password123
+    volumes:
+      - sqlserverdata:/var/opt/mssql
+    command: /bin/sh -c "(/opt/mssql/bin/sqlservr &) && sleep 10s && /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P Password123 -d master -q \"IF DB_ID('test_db') IS NULL CREATE DATABASE test_db\" && sleep infinity"
```

Comment:
Should we add a healthcheck that runs a `USE test_db` query?

I suspect this may be what's tripping up our CI. Because it's a bit of a slow machine, sqlserver is probably not running by the time the test runs.

Reply:
Yeah, fair. I'll make an entrypoint script to wait for sqlserver to be ready, something a little more robust.